contributing-guides/style-guide.md

```diff
@@ -325,6 +325,8 @@ For Windows command prompt, prepend and append the environment variable with a p
 
 Whereas for Powershell, prepend the environment variable with a dollar sign, Env and a colon, then enclose it with backticks (`$Env:VARIABLE_NAME`). For example: "Manage the `$Env:JAVA_HOME` environment variable".
 
+When describing file formats, either use the brand name like JSON, or prepend the file extensions with a dot like `.warts`.
```

Comment:
The "warts" example might be a little obscure. What about using an example that's both more well-known as a file extension, and looks more like a brand name in plain text?

```suggestion
When describing file formats, either use the brand name like JavaScript, or use a file extension with a dot like `.js`.
```

Reply:
It's not written anywhere, but we tend to always prefer brand names over technical terms. Having these be the same would go against that. 

I chose `.warts` only because I like to use examples where the guideline was discovered. We can change that to something else.

Reply:
> we tend to always prefer brand names over technical terms. Having these be the same would go against that.

I'm not sure I follow. Are you saying that the guideline you're adding currently says "Do X or Y", but you think it should imply "...but preferably X"?

> It's not written anywhere

Indeed, why not just say it explicitly, then? I don't think having Y refer to a different entity than X implies that we have a preference for X's form.